README.md

```diff
@@ -147,6 +147,26 @@ Once that's done, you can run the manager and API services:
     $ sudo esi-leap-api
 ```
 
+### Installation using Containerization
+
+By encapsulating the ESI Leap into a container, all the necessary dependencies, configurations and services are bundled into a single package.
+
+Please make sure to follow the instructions in the [Configuration](#configuration) section to generate the esi-leap.conf file, and ensure that it is present in the directory from which you are running the `podman build` command.
+
+Make sure to follow the instructions in the [Create Openstack Service](#create-the-openstack-service) to create the OpenStack service and endpoint. 
+
+After that build the container image using the podman build command, as Containerfile is located in the current directory. The syntax is as follows:
+
+```
+    $ podman build -t <image-name> -f Containerfile . 
+```
+
+Once the container image is built, you can run it using the podman run command. The syntax is as follows:
+
+```
+    $ podman run --name <container-name> -p <host-port>:<container-port> -d -i -t -v <host-directory>:<container-directory> <image-name> /bin/sh
+```
+
```

Comment:
This section should mention somewhere that the esi-leap.conf file must be mounted so that it's located at `/etc/esi-leap/esi-leap.conf` in the container. I'm also not sure what the purpose of the `-d -i -t` flags are in the sample command here-- they seem contradictory since `-d` is used to run the container in the background and `-i -t` are used to run the container in the current terminal. Finally, since there is a command to be run by default specified in the Containerfile, there should be no need to specify `/bin/sh` after `<image-name>`.